hedyorg/hedy
# 💻 Guest teacher connect option in teacher signup

Fixes #5428 

**How to test**
1. Go to the sign up page for teachers
2. Check the checkbox for connecting to a guest teacher
3. Make sure the phone number field pops up
4. Make sure only numbers are allowed in this field, not text.
<img width="1424" alt="Screenshot 2024-04-24 at 15 05 58" src="https://github.com/hedyorg/hedy/assets/48122190/9cc955f3-93b0-443c-9bed-8ec5476c04f4">
